src/compiler/tsc.ts

```diff
@@ -4,6 +4,10 @@
 module ts {
     var version = "1.4.0.0";
 
+    export interface SourceFile {
+        fileWatcher: FileWatcher;

```

Comment:
Because they're 1:1 related. In -watch mode every source file has a watcher attached and the natural place to store it is in the source file object. We do the same in services.ts for language service stuff.